src/ArrayValueCalculator.java

```diff
@@ -0,0 +1,33 @@
+public class ArrayValueCalculator {
+
+    public static String doCalc(String stringArray[][]) throws ArraySizeException, ArrayDataException {
+        int sum = 0;
```

Comment:
Would be better to declare this variable right after the validation, because it does not have any sense to allocate memory before validation is passed.